### PR TITLE
[VSLCV] HR Diagram

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-classifier

Describe your changes:
This adds the basic functionality for loading the subject image location for the HR diagram for the composite subject viewer. Here's how this looks in the story:

Screen Shot 2020-07-02 at 4 02 05 PM

Compared to the Invision:
https://projects.invisionapp.com/d/main?origin=v7#/console/17294981/403043950/preview?scrollOffset=210

I am having one problem with the accessibility. Since we know the image for this viewer will be a HR diagram, I was trying to label it, but VoiceOver keeps telling me it is not labeled, so I'm not sure what I'm doing wrong. I would appreciate any suggestions for this.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js [hunk 1]
```diff
@@ -108,20 +109,27 @@ const VariableStarViewer = React.forwardRef(function VariableStarViewer(props, r
...
         gridArea='HRDiagram'
+        margin='none'
         width='220px'
       >
         <SingleImageViewer
-          height={260}
+          aria-labelledby='imageId'
+          height={230}
           enableInteractionLayer={false}
+          role='image'
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js:123]: role=‘img’

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js [hunk 2]
```diff
@@ -108,20 +109,27 @@ const VariableStarViewer = React.forwardRef(function VariableStarViewer(props, r
...
+        as='figure'
         direction='column'
         height='260px'
         gridArea='HRDiagram'
+        margin='none'
         width='220px'
       >
         <SingleImageViewer
-          height={260}
+          aria-labelledby='imageId'
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js:120]: aria-label should work here too, if the SVG has the `img` role.

**Comment by srallen** [comment]
@beckyrother is it ok if those issues are addressed separately? The scope of this PR is to get the HR diagram image from the subject locations to display.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js (hunk 1 above)
Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js:123]: doh 🤦‍♀️

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js:123]: This was my issue. It's reading out as expected now.

**Comment by beckyrother** [comment]
@srallen yep, sorry!

**Comment by srallen** [comment]
@beckyrother no worries! I will open an issue to track the style issue in the other modules.

**Pull request merged**